src/node/miner.h

```diff
@@ -148,6 +150,7 @@ class BlockAssembler
     uint64_t nBlockSigOpsCost;
     CAmount nFees;
     std::unordered_set<Txid, SaltedTxidHasher> inBlock;
+    std::vector<FeeFrac> feerateHistogram;
```

Comment:
In commit "miner: `CreateNewBlock` will now track fee and virtual size of selected packages" (5171f16e6f96030fcc91b30bb196a2790e37848c)

It seems like it would be simpler and less fragile if `BlockAssembler::addPackageTxs` just populated `pblocktemplate->FeerateHistogram` directly, instead of filling this new `BlockAssembler::feerateHistogram` variable, which is effectively a temporary variable, and then moved from `BlockAssembler::feerateHistogram` to `pblocktemplate->FeerateHistogram`. It would be nice to be able to drop this new variable, and drop the new code moving the histogram from one variable to another.

Reply:
Good Suggestion.
Done!